packages/camera/lib/camera.dart

```diff
@@ -10,6 +11,127 @@ enum CameraLensDirection { front, back, external }
 
 enum ResolutionPreset { low, medium, high }
 
+typedef void OnLatestImageAvailable(CameraImage image);
+
+/// A single color plane of image data.
+///
+/// The number and meaning of the planes in an image are determined by the
+/// format of the Image.
+class Plane {
```

Comment:
nit: I would recommend splitting off these extra classes into their own files, and putting them into a src folder

Reply:
done